<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数字华容道</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/Blankcat11.png</normaloff>:/Blankcat11.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>392</width>
      <height>352</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="verticalSpacing">
      <number>2</number>
     </property>
    </layout>
   </widget>
   <widget class="QPushButton" name="backBtn2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>600</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;
iconSize:32 x 32</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/fast-backward-button.png</normaloff>:/fast-backward-button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>阿里妈妈东方大楷</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>布局大小：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="ChosenWidth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>阿里妈妈东方大楷</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 2px solid #f3f3f3;/*设置线宽*/
font: 12pt &quot;阿里妈妈东方大楷&quot;;
background-color: rgb(237, 242, 255);/*背景颜色*/

	border-color:rgb(77, 123, 255);/*边框颜色*/

border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
/*QCombobox右侧按钮*/
CbWidth::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;/*放于右方顶部*/
    border-top-right-radius: 3px;/*设置边框圆角*/
    border-bottom-right-radius: 3px;
	width:50px
/*padding-right: 50px;*/
}
/*QCombobox右侧按钮的箭头图标*/

CbWidth::down-arrow
{
	width:16px;
	height:16px;
	border-image: url(:/plain-arrow.png);
}
CbWidth QAbstractItemView::item {
	border-radius: 15px;/*圆角*/
    height: 30px;   /* 项的高度（设置pComboBox-&gt;setView(new QListView());后，该项才起作用） */
	background-color: rgb(237, 242, 255);

}

</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>2x2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3x3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4x4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5x5</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnRandom">
       <property name="styleSheet">
        <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;
iconSize:32 x 32</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/RECHANGE.png</normaloff>:/RECHANGE.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="BtnReset">
       <property name="styleSheet">
        <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;
iconSize:32 x 32</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/RESTART.png</normaloff>:/RESTART.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>阿里妈妈东方大楷</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>最少次数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabMin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>阿里妈妈东方大楷</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>阿里妈妈东方大楷</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>操作次数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabCount">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>15</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>阿里妈妈东方大楷</family>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>520</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="BtnNextStep">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style:none;
padding:11px;
border-radius:3px;
background:transparent;
iconSize:32 x 32</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/next.png</normaloff>:/next.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="GameFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>390</width>
      <height>350</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>350</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>阿里妈妈东方大楷</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color:rgb(216, 255, 252);
	border : 2px solid gray;
    border-radius: 5px;
	padding : 2px;
	margin : 2px;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item row="0" column="0">
      <layout class="QGridLayout" name="GameGrid">
       <property name="spacing">
        <number>10</number>
       </property>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
